gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineEclipseIntegrationTest.groovy

```diff
@@ -20,7 +20,7 @@ import com.google.common.base.Charsets
 import com.google.common.io.Files
 import nebula.test.IntegrationSpec
 import nebula.test.functional.ExecutionResult
-import org.apache.commons.io.FileUtils
+import org.gradle.internal.impldep.org.apache.commons.io.FileUtils
```

Comment:
yes, I was just going to suggest that. can we stay on gradle 2.13 for a bit too, please? (don't want people to upgrade unnecessarily often.)